washingtonpost/elex-clarity
# Improvements to election settings handling plus, you know, rewriting lots of stuff

## Description
We need to be able to create a mapping from race IDs to last updated timestamps using [Clarity's electionsettings.json](https://results.enr.clarityelections.com/GA/105369/270998/json/en/electionsettings.json). This PR adds a converter for such settings and does 50 other things because why not! Just kidding; I know I have a problem. I will update with the list of changes soon, just wanted to get this up before Anthony goes on vacation (sorry @anthonyjpesce ☹️ ).

## Jira Ticket

## Test Steps
Basically try all the examples in the readme and make sure they still work! The new settings functionality in particular requires that you run `elexclarity 105369 GA --outputType=settings --officeID=P --countyMapping=<full actual county mapping>`.
